account_vat_period_end_statement/__init__.py

```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
 #
```

Comment:
Please fix copyrights following the format defined in the [template module](https://github.com/OCA/maintainer-tools/tree/master/template/module): generic in `__init__` files, with authors elsewhere (xml files too)